packages/fiori/test/specs/SideNavigation.spec.js

```diff
@@ -141,5 +141,22 @@ describe("Component Behavior", () => {
 			// clean up
 			await browser.$("#sn1").setProperty("collapsed", false);
 		});
+	
+		it("tests the prevention of the ui5-selection-change event", async () => {
+			const selectionChangeCheckbox = await browser.$("#prevent-selection");
+			await selectionChangeCheckbox.click();
+
+			const selectionChangeCounter = await browser.$("#counter");
+			await selectionChangeCounter.setProperty("value", "0");
+
+			const sideNavigation = await browser.$("#sn1");
+			const items = await sideNavigation.shadow$("ui5-tree").shadow$("ui5-list").$$("ui5-li-tree");
+			await items[0].click();
+			await items[1].click();
+
+			assert.strictEqual(await selectionChangeCounter.getProperty("value"), "0", "Event is not fired");
```

Comment:
I find "Event is not fired" is somehow misleading, no. It's fired, but it's prevented. 
Perhaps we need to check if the old selection remains and the new one is not applied when the user performs clicks.

Reply:
You are right. I've changed the test.